projects/fizzbuzz/java/FizzBuzz.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * This is a function to iterate 100 times, if the iteration
+ * is divisible by 3 or 5, print Fizz, or Buzz respectivly, 
+ * else print the number of the iteration starting at 1, ending at 100.
+ */
+package fizzbuzz;
+
+/**
+ *
+ * @author visionaryJack
+ */
+public class FizzBuzz {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        String fizz = "Fizz"; //String for 3

```

Comment:
There is one extra space on this line before the word "String". The declaration of String variables of fizz and buzz could be eliminated by incorporating them directly into lines: 25, 29, 33 if total file size is a concern for your application of this program.